### PR TITLE
Order update action set customer

### LINKED ISSUE #1791: Add new Action to allow users to change customer in order
Backend follow up ticket

### PR DESCRIPTION
**Pull request by acbeni**
# Description

Closes #1791

# Checklist

- [x] Update release Notes
- [x] Add to the current github milestone

### CODE REVIEW

File: sdk-http-apache-async/src/test/java/io/sphere/sdk/http/Tls12Test.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+import javax.net.ssl.SSLContext;
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+
+public class Tls12Test {
```
Comment (katmatt) [inline sdk-http-apache-async/src/test/java/io/sphere/sdk/http/Tls12Test.java]: May be we should make this part of another PR?

Comment (acbeni) [reply sdk-http-apache-async/src/test/java/io/sphere/sdk/http/Tls12Test.java]: Yeah, this should be part of a diff PR, would remove it 👍

**Comment by acbeni** [comment]
@katmatt fixed code review 👍

**Comment by katmatt** [comment]
Perfect 👍
